pkg/kn/commands/service/service_create.go

```diff
@@ -134,6 +134,9 @@ func flush(out io.Writer) {
 func createService(client v1alpha1.KnClient, service *serving_v1alpha1_api.Service, namespace string, out io.Writer) error {
 	err := client.CreateService(service)
 	if err != nil {
+		if api_errors.IsNotFound(err) {
```

Comment:
Fixes for service create, but other commands would still report the error mentioned in issue, I think this needs to be fixed  in more generic way which applies to other commands as well.

Reply:
I could implement it in the `KnClient` level for each method if that makes sense

Reply:
Good idea. Maybe rephrase to a shorter sentence like (and to put knative later in the sentence to allow the proper capitalization):

```
no Knative serving API found on the backend. Please verify the installation.
``

open for other suggestions, though. I wonder whether we should also wrap the original error ? But I think there is no extra context information in that error which goes beyond that information.

Reply:
I thought about wrapping it into a custom error struct but there is not a lot of information returned by the original error struct itself besides the error message.. could still be useful in the future, idk :)